src/NzbDrone.Core/NetImport/TMDb/TMDbRequestGenerator.cs

```diff
@@ -72,6 +79,7 @@ public virtual NetImportPageableRequestChain GetMovies()
                 requestBuilder.Resource(tmdbParams);
 
                 var request = requestBuilder
+                    .SetHeader("Bearer","NEED RADARR API Read Access Token (v4 auth) HERE")
```

Comment:
Maybe I misunderstood the documentation, but isn't this specific to each user? So wouldn't this need to come from the user?

Reply:
Also, if this is not the case, wouldn't we still need an access_token for each user to get those lists? Something we can only do by redirecting them to the TMDB homepage?

Reply:
No that one is the generic Radarr one, only the V4 version. Each user could use its own for sure but the way it's set up currently for v3 in radarr is everyone uses the radarr one. Guess it makes it easier for you to track usage stats etc.

Reply:
API token is `eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIxYTczNzMzMDE5NjFkMDNmOTdmODUzYTg3NmRkMTIxMiIsInN1YiI6IjU4NjRmNTkyYzNhMzY4MGFiNjAxNzUzNCIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.gh1BwogCCKOda6xj9FRMgAAj_RYKMMPC3oNlcBtlmwk`